frontend/src/pages/Collaboration.js

```diff
@@ -3,26 +3,23 @@ import { useLocation, useNavigate } from 'react-router-dom';
 import io from 'socket.io-client';
 import { Chat, CodeEditor, QuestionPanel } from '../components/Collaboration';
 import { QuestionContent } from '../components/Question';
+import { ChangeQuestionWindow } from '../components/ConfirmationWindows';
```

Comment:
Perhaps reorder them by `../components/*`, i.e.
```js
import { Chat, CodeEditor, QuestionPanel } from '../components/Collaboration';
import { ChangeQuestionWindow } from '../components/ConfirmationWindows';
import { QuestionContent } from '../components/Question';
```

Reply:
Dont get what you meant by this

Reply:
Basically for these imports, order them alphabetically based in the `from ...` part.

So `'../components/Collaboration'` then `'../components/ConfirmationWindows'` then `'../components/Question'`.